src/OrbitQt/orbitdataviewpanel.cpp

```diff
@@ -9,64 +9,61 @@
 #include <QPushButton>
 #include <string>
 
-#include "orbittablemodel.h"
 #include "ui_orbitdataviewpanel.h"
 
 OrbitDataViewPanel::OrbitDataViewPanel(QWidget* parent)
-    : QWidget(parent), ui(new Ui::OrbitDataViewPanel) {
-  ui->setupUi(this);
-  ui->label->hide();
+    : QWidget(parent), ui_(new Ui::OrbitDataViewPanel) {
+  ui_->setupUi(this);
+  ui_->label->hide();
 }
 
-OrbitDataViewPanel::~OrbitDataViewPanel() { delete ui; }
+OrbitDataViewPanel::~OrbitDataViewPanel() { delete ui_; }
 
 void OrbitDataViewPanel::Initialize(orbit_data_views::DataView* data_view,
                                     SelectionType selection_type, FontType font_type,
                                     bool uniform_row_height,
                                     QFlags<Qt::AlignmentFlag> text_alignment) {
-  ui->treeView->Initialize(data_view, selection_type, font_type, uniform_row_height,
-                           text_alignment);
+  ui_->treeView->Initialize(data_view, selection_type, font_type, uniform_row_height,
+                            text_alignment);
 
-  std::string label = ui->treeView->GetLabel();
+  std::string const label = ui_->treeView->GetLabel();
```

Comment:
Interesting const placement

I dont know if the style guide actually enforces this, but in [this](https://google.github.io/styleguide/cppguide.html#Constant_Names) section it has is `const type`

```suggestion
  const std::string label = ui_->treeView->GetLabel();
```

Reply:
I like that better as well, visual studio did it wrong :) done.